src/main/java/by/lawaksoft/tradebot/service/util/impl/CreateTradeMessageServiceImpl.java

```diff
@@ -0,0 +1,52 @@
+package by.lawaksoft.tradebot.service.util.impl;
+
+import by.lawaksoft.tradebot.dto.amend_order.AmendOrderRequestDTO;
+import by.lawaksoft.tradebot.dto.cancel_order.CancelOrderRequestDTO;
+import by.lawaksoft.tradebot.dto.place_order.PlaceOrderRequestDTO;
+import by.lawaksoft.tradebot.entity.enums.Method;
+import by.lawaksoft.tradebot.mapper.JsonMapper;
+import by.lawaksoft.tradebot.service.util.CreateTradeMessageService;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CreateTradeMessageServiceImpl implements CreateTradeMessageService {
+
+    private final static String ORDER_PATH = "/api/v5/trade/order";
+    private final static String CANCEL_ORDER_PATH = "/api/v5/trade/cancel-order";
+    private final static String AMEN_ORDER_PATH = "/api/v5/trade/amend-order";
```

Comment:
could you move this variable values into properties file

Reply:
done